templates/types/streaming/fastapi/app/api/controllers/llama_cloud.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+from typing import Any, Dict, List, Optional
+
+import requests
+from app.api.routers.models import LlamaCloudFile
+
+logger = logging.getLogger("uvicorn")
+
+
+class LLamaCloudFileController:
```

Comment:
@leehuwuj still looks like a stateless service class to me, so why not `LLamaCloudFileService`?